source/skins/simple/CheckboxSkin.js

```diff
@@ -35,6 +35,7 @@ export const CheckboxSkin = (props: Props) => (
   >
     <input
       {...pickDOMProps(props)}
+      onChange={() => {}}
```

Comment:
Could you please use some kind of omit (spread or lodash omit) instead of this override? It doesn't show clear intentions and looks like a leftover. 😅